searx/__init__.py

```diff
@@ -60,3 +60,49 @@
     settings['server']['secret_key'] = environ['SEARX_SECRET']
 if 'SEARX_BIND_ADDRESS' in environ:
     settings['server']['bind_address'] = environ['SEARX_BIND_ADDRESS']
+
+
+class _brand_namespace:
+
+    @classmethod
+    def get_val(cls, group, name, default=''):
+        return settings[group].get(name, False) or ''
```

Comment:
should be `return settings.get(group, {}).get(name) or default`:
* to use `default`.
* to be fail safe in `group` doesn't exist in the settings.

see commit https://github.com/dalf/searx/commit/1074ad778071ca5208932d16e909fcd8dc811777

Reply:
I cherry picked 1074ad77 / thanks!